app/models/work_package/pdf_exporter.rb

```diff
@@ -363,8 +363,17 @@ def textstring(s)
       end
     end
 
-    alias RDMCell Cell
-    alias RDMMultiCell MultiCell
+    def RDMCell(w,h=0,txt='',border=0,ln=0,align='',fill=0,link='')
+        Cell(w,h,fix_text_encoding(txt),border,ln,align,fill,link)
+    end
+
+    def RDMMultiCell(w,h=0,txt='',border=0,align='',fill=0)
```

Comment:
Use snake_case for method names.<br>Space missing after comma.<br>Surrounding space missing in default value assignment.